src/containers/Tenant/Diagnostics/TenantOverview/TenantStorage/i18n/en.json

```diff
@@ -0,0 +1,4 @@
+{
+  "alert_no-data": "No data",
+  "context_capacity-usage": "{{value}} of {{capacity}}"
+}
```

Comment:
It is and src/components/ProgressViewer/i18n/en.json are duplicates

Reply:
key in ProgressView is unused - removed it